sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v2/query.sql

```diff
@@ -0,0 +1,83 @@
+/*
+
+This query is run in a standalone project and is scheduled separately from our
+normal Airflow infrastructure, but is codified here for discoverability and for
+making use of the testing harness here.
+
+Note that we need to fully qualify all table references since this runs from
+a separate project.
+
+*/
+WITH impressions AS (
+  SELECT
+    -- This should already be truncated to second level per CONSVC-1364
+    -- but we reapply truncation to be explicit about granularity.
+    TIMESTAMP_TRUNC(submission_timestamp, SECOND) AS submission_timestamp,
+    request_id,
+    search_query AS telemetry_query,
+    advertiser,
+    block_id,
+    context_id,
+    sample_id,
+    is_clicked,
+    locale,
+    metadata.geo.country,
+    metadata.geo.subdivision1 AS region,
+    normalized_os,
+    normalized_os_version,
+    release_channel,
+    position,
+    reporting_url,
+    scenario,
+  FROM
+    `moz-fx-data-shared-prod.contextual_services_stable.quicksuggest_impression_v1`
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+),
+merino_logs AS (
+  SELECT
+    TIMESTAMP_TRUNC(timestamp, SECOND) AS merino_timestamp,
+    jsonPayload.fields.rid AS request_id,
+    jsonPayload.fields.query,
+    NULLIF(jsonPayload.fields.country, 'none') AS merino_country,
+    NULLIF(jsonPayload.fields.region, 'none') AS merino_region,
+    NULLIF(jsonPayload.fields.dma, 'none') AS merino_dma,
+    -- -- We are not propagating city-level data to sanitized table.
+    -- NULLIF(jsonPayload.fields.city, 'none') AS merino_city,
+    NULLIF(jsonPayload.fields.form_factor, 'none') AS merino_form_factor,
+    NULLIF(jsonPayload.fields.browser, 'none') AS merino_browser,
+    NULLIF(jsonPayload.fields.os_family, 'none') AS merino_os_family,
```

Comment:
does merino have a version that we would want to include here? @ncloudioj (couldn't tag mythmon, not sure why)

Reply:
Merino does have a version, but I don't think we're reporting it in the operational log now. We can add it if that's preferred. 

Hmm, I can't tag mythmon either.

Reply:
ok - including a version would be helpful if you want to experiment with different suggestion logic in merino and see which ones were clicked on more.